core/src/main/java/com/coderanch/blackjack/Cards.java

```diff
@@ -61,4 +64,22 @@ private Cards() {}
     static Set<Card> getStandardDeck() {
         return STANDARD_DECK;
     }
+
+    static Stack<Card> getShuffledStandardDeck(){
+        return getStandardDeck().stream()
+                .map(c -> new CardPair(RANDOM.nextInt(10_000_000), c))
+                .sorted((o1, o2) -> Integer.compare(o1.position, o2.position))
+                .map(cp -> cp.card)
+                .collect(Collectors.toCollection(Stack::new));
+    }
+
+    private static class CardPair{
+        public final int position;
+        public final Card card;
+
+        private CardPair(int position, Card card) {
+            this.position = position;
+            this.card = card;
+        }
+    }
```

Comment:
I am pleased that you used a new private class to simplify a functional operation, but if you use my suggestion above, you won't need it anymore.